bin/prettier.js

```diff
@@ -193,7 +193,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log(chalk.grey("%s %dms"), filename, Date.now() - start);
```

Comment:
I add this after this comment https://github.com/jlongster/prettier/pull/584#discussion_r99375766
> even though this has been ignored, it still took time to actually prettify that file